src/main/java/net/neoforged/neoforge/common/data/internal/NeoForgeLanguageProvider.java

```diff
@@ -294,6 +298,10 @@ protected void addTranslations() {
         add(Tags.Items.SANDSTONE_SLABS, "Sandstone Slabs");
         add(Tags.Items.SANDSTONE_STAIRS, "Sandstone Stairs");
         add(Tags.Items.SANDSTONE_RED_BLOCKS, "Red Sandstone Blocks");
+        add(Tags.Items.PUMPKINS, "Pumpkins");
+        add(Tags.Items.PUMPKINS_NORMAL, "Normal Pumpkins");
+        add(Tags.Items.PUMPKINS_CARVED, "Carved Pumpkins");
+        add(Tags.Items.PUMPKINS_JACK_O_LANTERNS, "Jack o'Lanterns");
```

Comment:
This should be moved to the proper position with respect to the alphabetical order of tag names.